plugins/auth-backend/src/service/router.ts

```diff
@@ -67,6 +71,28 @@ export async function createRouter(
 
   const appUrl = config.getString('app.baseUrl');
   const authUrl = await discovery.getExternalBaseUrl('auth');
+  const cacheMemoryType = config.getString('backend.cache.store');
+  let cacheStore = null;
+
+  switch (cacheMemoryType) {
+    case 'redis':
+      {
+        const redisClient = await getRedisClient({
+          redisConnectionUrl: config.getString('backend.cache.connection'),
```

Comment:
This will most likely need a more thought out connection block as users are likely going to configure other parameters as well. url should for example likely be a part of that block and not be connection.

Reply:
actually, In URL, we can pass all the configuration parameters too. therefore I went forward with this